### ISSUE TITLE
BasicContext#loadLocale error

### ISSUE THREAD

**Issue by guai** [impossibl/pgjdbc-ng#328]
I think this lines:
```java
      // Check if locale matches a win32 type locale (e.g. "English_United States.1252")
      if (Locales.getJavaCompatibleLocale(localeSpec.split("\\.")[0]) != null) {
        localeSpec = Locales.getJavaCompatibleLocale(localeSpec);
      }
```
should be changed to this:
```
      // Check if locale matches a win32 type locale (e.g. "English_United States.1252")
	String priorDot = localeSpec.split("\\.")[0];
      if (Locales.getJavaCompatibleLocale(priorDot) != null) {
        localeSpec = Locales.getJavaCompatibleLocale(priorDot);
      }
```
cause `Locales.getJavaCompatibleLocale` awaits something like `Russian_Russia` and not `Russian_Russia.1251`

**Comment by guai** [comment]
version 0.7.1

**Comment by ricpdx** [comment]
I can see that this fix is in the source code here on github -- but it is not in the 0.7.1 artifacts on maven central and jcentral for some reason.

**Comment by ricamn** [comment]
Still not fixed.

**Comment by ricamn** [comment]
Not fixed yet.

**Comment by atmaxinger** [comment]
Is there an ETA when this will be available in Maven?

**Comment by derroman** [comment]
We definitely need this asap, please!

**Comment by kdubb** [comment]
Fixed in #349 (and possibly earlier)
